src/components/menu/menu.css

```diff
@@ -27,6 +27,12 @@
     line-height: 34px;
     white-space: nowrap;
     padding: 0 10px;
+
+    /* these are needed to reset styles that are inherited from frameless.scss
+    when embedded in scratch-www */
+    font-size: .75rem;
+    font-weight: bold;
+    margin: 0;
```

Comment:
We should do this in www not here.

Reply:
Hm, not sure about this one. Shouldn't this component be in charge of its own text and margin/padding style, so that it appears correctly no matter what top level tag styles were set globally?

Reply:
(Also, I'm not sure what other alternative we have, besides duplicating various css from gui in www, and having www pass it back into gui.)

Reply:
For now let's remove this, we can address it as a part of https://github.com/LLK/scratch-www/issues/2096 (this comment will create a reference there)

Reply:
OK. FYI, this currently makes all the gui menus (File, Edit, etc.) have a larger font size (1rem) than they're supposed to (.75rem).